### PR TITLE
MSDKUI-2155: Support Gradle 6

### PR DESCRIPTION
**Pull request by tstaszak89**
Projects now supports gradle 6.0.1 and gradle plugin 3.5.3
FindBugs static analysis tool is deprecated in gradle 6.0.1 so it has
been replaced by SpotBugs.

Signed-off-by: Tadeusz Staszak <36914750+tstaszak89@users.noreply.github.com>

### CODE REVIEW

File: MSDKUIDev/gradlew [hunk 1]
```diff
@@ -28,7 +28,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m"'
```
Comment (stefan-troyke) [inline MSDKUIDev/gradlew]: Please correct.

File: MSDKUIKit/coverage.gradle [hunk 2]
```diff
@@ -1,7 +1,23 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
```
Comment (stefan-troyke) [inline MSDKUIKit/coverage.gradle]: wrong copyright header, please let me know if you split this in a different commit.

File: MSDKUIDev/gradlew (hunk 1 above)
Comment (tstaszak89) [reply MSDKUIDev/gradlew]: It is intentional. In all three projects, gradle version was increased using "./gradlew wrapper --gradle-version 6.0.1" command. Execution of this command changed also those gradlew files. Additionally later in this bash script, variable DEFAULT_JVM_OPTS is passed as argument for "eval set" which prepares it to be an option for command execution and quote characters are needed to indicate that this is s single command.

File: MSDKUIKit/coverage.gradle (hunk 2 above)
Comment (tstaszak89) [reply MSDKUIKit/coverage.gradle]: Yes, I will update copyright headers in entire repository in different commit.
